hack/patches/006-version.patch

```diff
@@ -5,30 +5,13 @@ index aef33927e..585d66fdf 100644
 @@ -44,8 +44,10 @@ func (c *ClientConfig) InitFlags(fs *flag.FlagSet) {
  	fs.StringVar(&c.ServerURL, "server", "",
  		"The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
- 
+
 -	fs.StringVar(&c.Kubeconfig, "kubeconfig", os.Getenv("KUBECONFIG"),
 -		"Path to a kubeconfig. Only required if out-of-cluster.")
 +	if fs.Lookup("kubeconfig") == nil {
 +		fs.StringVar(&c.Kubeconfig, "kubeconfig", os.Getenv("KUBECONFIG"),
 +			"Path to a kubeconfig. Only required if out-of-cluster.")
 +	}
- 
+
  	fs.IntVar(&c.Burst, "kube-api-burst", int(envVarOrDefault("KUBE_API_BURST", 0)), "Maximum burst for throttle.")
- 
-diff --git a/vendor/sigs.k8s.io/controller-runtime/pkg/client/config/config.go b/vendor/sigs.k8s.io/controller-runtime/pkg/client/config/config.go
-index fd2772412..54aab6c3d 100644
---- a/vendor/sigs.k8s.io/controller-runtime/pkg/client/config/config.go
```

Comment:
AFAIK this is no longer needed. But it might be good if someone can take another look.